doc/auto-discovery-docker.md

```diff
@@ -89,51 +78,38 @@ Then user can follow [the doc to add Provision Watcher](./auto-discovery.md).
 
 #### 2. Replace the `add-device-onvif-camera.yml` with the following content:
 ```yaml
+version: '3.7'
+
 services:
   device-onvif-camera:
-    image: edgex/device-onvif-camera${ARCH}:${DEVICE_ONVIFCAM_VERSION}
-    user: "${EDGEX_USER}:${EDGEX_GROUP}"
+    image: edgexfoundry/device-onvif-camera${ARCH}:${DEVICE_ONVIFCAM_VERSION}
+    ports:
+      - "127.0.0.1:59984:59984"
     container_name: edgex-device-onvif-camera
     hostname: edgex-device-onvif-camera
     read_only: true
     restart: always
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+      - device-common.env
     environment:
       SERVICE_HOST: edgex-device-onvif-camera
-      EDGEX_SECURITY_SECRET_STORE: "true"
-      DEVICE_DISCOVERY_ENABLED: "true"
-      DRIVER_DISCOVERYETHERNETINTERFACE: enp0s3
-      DRIVER_DEFAULTSECRETPATH: credentials001
-      
+      MESSAGEQUEUE_HOST: edgex-redis
       SECRETSTORE_HOST: localhost
       STAGEGATE_BOOTSTRAPPER_HOST: localhost
       STAGEGATE_READY_TORUNPORT: 54329
       STAGEGATE_WAITFOR_TIMEOUT: 60s
-
-      WRITABLE_LOGLEVEL: DEBUG
     depends_on:
       - consul
       - data
       - metadata
     security_opt:
       - no-new-privileges:true
-    command: /device-onvif-camera -cp=consul.http://localhost:8500 --registry --confdir=/res
+    user: "${EDGEX_USER}:${EDGEX_GROUP}"
 ```
 
-**Note**: The user should replace the host IP to match their own machine IP
-
-- Remove the useless port mapping when using the host network
-- Replace **networks** with **network_mode: "host"**
-- Remove `env_file` because we don't use the env like `CLIENTS_CORE_DATA_HOST=edgex-core-data`
-- Modify `SERVICE_HOST` env to match the machine IP
-- Add `EDGEX_SECURITY_SECRET_STORE` env with "true" value
-- Enable auto-discovery by `DEVICE_DISCOVERY_ENABLED` with "true" value
-- Use `DRIVER_DISCOVERYETHERNETINTERFACE` to specify the ethernet interface for discovering
-- Use `DRIVER_DEFAULTSECRETPATH` to specify the default secret path
-- Use `SECRETSTORE_HOST` to specify the Vault's host
-- Use `STAGEGATE_BOOTSTRAPPER_HOST`, `STAGEGATE_READY_TORUNPORT`, `STAGEGATE_WAITFOR_TIMEOUT` to specify the bootstrapper settings for waiting the security set up
-- Use `WRITABLE_LOGLEVEL` to specify the log level for debugging
-- Add command to override the CMD because we don't use the configuration provider from Consul
-
 #### 3. Export the Security Bootstrapper
```

Comment:
Why is this being done?

Reply:
Not sure what you are referring to here?

Reply:
`Export the Security Bootstrapper`

Reply:
I think this was for running hybrid mode??  Compose builder will generate the compose file so this isn't needed, so yea it can be part of #69. I'll add it to the description.